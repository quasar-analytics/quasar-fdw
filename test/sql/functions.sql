PREPARE statepop(char(2)) AS SELECT sum(pop) FROM zips WHERE state = $1;
EXECUTE statepop('CO');
EXECUTE statepop('RI');
EXPLAIN (COSTS off) EXECUTE statepop('MA');
DEALLOCATE statepop;
