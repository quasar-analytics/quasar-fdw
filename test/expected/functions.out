PREPARE statepop(char(2)) AS SELECT sum(pop) FROM zips WHERE state = $1;
EXECUTE statepop('CO');
   sum   
---------
 3293755
(1 row)

EXECUTE statepop('RI');
   sum   
---------
 1003218
(1 row)

EXPLAIN (COSTS off) EXECUTE statepop('MA');
                               QUERY PLAN                                
-------------------------------------------------------------------------
 Aggregate
   ->  Foreign Scan on zips
         Quasar query: SELECT "pop" FROM "zips" WHERE (("state" = 'MA'))
(3 rows)

DEALLOCATE statepop;
